src/components/AssetHeader/AssetChart.tsx

```diff
@@ -65,9 +66,15 @@ export const AssetChart = ({ accountId, assetId, isLoaded }: AssetChartProps) =>
   const { price } = marketData || {}
   const assetPrice = toFiat(price) ?? 0
   const [view, setView] = useState(accountId ? View.Balance : View.Price)
-  const filter = useMemo(() => ({ assetId, accountId }), [assetId, accountId])
   const translate = useTranslate()
 
+  const accountSpecifiers = useAppSelector(state => selectAccountSpecifier(state, asset?.caip2))
+  const accountSpecifier = accountSpecifiers?.[0] // TODO: maybe remove me, or maybe not
+  const filter = useMemo(
+    () => ({ assetId, accountId, accountSpecifier }),
+    [assetId, accountId, accountSpecifier],
```

Comment:
Not 100% sure about this one. It could potentially cause regressions - it's not wrong to use `accountSpecifier`, in fact we do need it for Cosmos,